docs/content-editing/add-a-file.md

```diff
@@ -0,0 +1,172 @@
+---
+myst:
+  html_meta:
+    "description": "Add a File"
+    "property=og:description": "Add a File"
+    "property=og:title": "Add a File"
+    "keywords": "Plone, Add, File"
+---
+
+# Add a File
+
+You can add any type of file to your Plone site. Files can be any "binary file", including PDFs, text files, videos, and Word documents. 
+
+You could follow this method to add an image, but rather than add an image as a file, you should add an image as an image! (see Add an Image below).
+
+## Add a File
+
+[//]: # (#### [Made by T. Kim Nguyen with Scribe]&#40;https://scribehow.com/shared/Add_a_File__Yh-Avk1hQwatZwoP5HAGkA&#41;)
```

Comment:
Use proper comment syntax. `[//]: #` causes the following warning. Globally replace with `%`. This change is mandatory to remove noisy warnings during builds.

```console
docs/content-editing/add-a-link.md:18: WARNING: Duplicate reference definition: // [myst.duplicate_def]
```

```suggestion
% (#### [Made by T. Kim Nguyen with Scribe]&#40;https://scribehow.com/shared/Add_a_File__Yh-Avk1hQwatZwoP5HAGkA&#41;)
```

Reply:
somehow I wasn't able to get that % syntax to work, so I deleted all the commented lines...

Reply:
What did you try that did not work? The suggestion works for me. See https://myst-parser.readthedocs.io/en/latest/syntax/typography.html#comments